content/games/factions/companies.md

```diff
@@ -73,6 +80,12 @@ Vicious (3)
 Well-Armed (1)
 : Once each turn, may reroll all results of 1 on a single to-hit roll.
 
+| Taunt                 | Check | Requirements                                                                     | Effect                                                                                                                                                                                                                                           |
+| :-------------------- | :---: | :------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Blood Makes Mud!      |  6+   | Shieldkin only. Must be within 6" of a melee.                                    | When this Group Attacks an enemy already in melee, they roll three extra dice. While in melee, this Group is Unfeeling.                                                                                                                          |
+| Our Bolts, Your Eyes! |  7+   | Mounted Crossbows only. Must be within 12" of an enemy.                          | This Group becomes Reckless, except they must Shoot instead of Attack if they are able. They only hit on 6 when Shooting and each hit counts twice.                                                                                              |
```

Comment:
```suggestion
| Our Bolts, Your Eyes! |  7+   | Mounted Crossbows only. Must be within 12" of an enemy.                          | This Group becomes Reckless, except they must Shoot instead of Attack, if able. They only hit on 6 when Shooting, and each hit counts twice.                                                                                              |
```